examples/lighting-app/bouffalolab/bl602/include/CHIPProjectConfig.h

```diff
@@ -149,7 +149,7 @@
  *
  * Enable recording UTC timestamps.
  */
-#define CHIP_CONFIG_EVENT_LOGGING_UTC_TIMESTAMPS 1
+//#define CHIP_CONFIG_EVENT_LOGGING_UTC_TIMESTAMPS 1
```

Comment:
Perhaps just delete this, rather than comment?

Reply:
Thanks for pointing it out, I have modified it as you said.